.github/workflows/smoke-test-pr-check.yaml

```diff
@@ -0,0 +1,156 @@
+          REPLACE_FROM="COPY --from=linux-musl-content --chown=0:0 /checode-linux-musl /mnt/rootfs/checode-linux-musl"
+          sed -i -r -e "s|${REPLACE_FROM}||" build/dockerfiles/assembly.copy.Dockerfile
+          export DOCKER_BUILDKIT=1
+          docker build -f build/dockerfiles/assembly.copy.Dockerfile -t che-code .
+          docker tag che-code:latest che-code:next
+          docker save -o /tmp/che-code-latest.tar che-code:latest che-code:next
+
+      - name: Start minikube
+        id: run-minikube
+        uses: che-incubator/setup-minikube-action@next
+        with:
+          minikube-version: v1.23.2
+
+      - name: Install chectl
+        run: bash <(curl -sL https://www.eclipse.org/che/chectl/) --channel=next
+
+      - name: Deploy Che
+        run: |
+          #
+          # load che-code images from /tmp/che-code-latest.tar
+          #
+          eval $(minikube docker-env) && docker load -i  /tmp/che-code-latest.tar && rm /tmp/che-code-latest.tar
+
+          #
+          # deploy Che
+          #
+          chectl server:deploy \
+            --batch \
+            --platform minikube \
+            --k8spodwaittimeout=120000 \
+            --k8spodreadytimeout=120000 \
+            --che-operator-cr-patch-yaml "${GITHUB_WORKSPACE}/build/test/github-minikube-checluster-patch.yaml"
+
+          #
+          # appy patch
+          #
+          kubectl patch devworkspaceoperatorconfigs \
+            -n eclipse-che devworkspace-config \
+            --patch '{"config": {"workspace": {"imagePullPolicy": "IfNotPresent"}}}' \
+            --type merge
+
+      - name: Pull Universal Base Image
+        run: |
+          minikube image pull quay.io/devfile/universal-developer-image:ubi8-latest
+
+      - name: Run Smoke test
+        run: |
+          echo "> Local test dir: ${LOCAL_TEST_DIR}"
+          docker run \
+            --shm-size=2048m \
+            -p 5920:5920 \
+            --network="host" \
+            -e TS_SELENIUM_LOAD_PAGE_TIMEOUT=60000 \
+            -e TS_SELENIUM_USERNAME=che@eclipse.org \
+            -e TS_SELENIUM_PASSWORD=admin \
+            -e TS_SELENIUM_BASE_URL=https://$(kubectl get ingress che -n eclipse-che -o jsonpath='{.spec.rules[0].host}') \
+            -e DELETE_WORKSPACE_ON_FAILED_TEST=true \
+            -e TS_SELENIUM_START_WORKSPACE_TIMEOUT=120000 \
+            -e NODE_TLS_REJECT_UNAUTHORIZED=0 \
+            -e VIDEO_RECORDING=true \
+            -e TS_SELENIUM_LOG_LEVEL=TRACE \
+            -e TS_WAIT_LOADER_PRESENCE_TIMEOUT=120000 \
+            -e TS_COMMON_DASHBOARD_WAIT_TIMEOUT=30000 \
+            -v ${LOCAL_TEST_DIR}/tests/e2e/report:/tmp/e2e/report:Z \
+            -v ${LOCAL_TEST_DIR}/tests/e2e/video:/tmp/ffmpeg_report:Z \
+            -e USERSTORY=SmokeTest \
+            -e TS_SELENIUM_VALUE_OPENSHIFT_OAUTH=false \
+            -e TEST_REPO=https://github.com/vitaliy-guliy/quarkus-api-example \
+            quay.io/mmusiien/che-e2e:smoke-test
```

Comment:
Are there specific changed in quay.io/mmusiien/che-e2e:smoke-test image compare to quay.io/eclipse/che-e2e image built from https://github.com/eclipse/che main branch?

Reply:
@musienko-maxim could you comment here?

Reply:
Yes in our image login to k8s is not implemented and we need to improve it. We need to add changes to be able to run on different platforms and add this to https://github.com/eclipse/che. The custom image was built by my local machine

Reply:
@dmytro-ndp @musienko-maxim  what can we do to improve tests in **eclipse/che** to avoid using of developer repository ( quay.io/mmusiien ) ?

Reply:
So, we need to create an issue because this will affect not on the login process but also all  API calls (because if we log in to minikube the authorization token that we grab from the browser will be different). After implementing this we need to make sure that it works properly on different platforms.  And I guess it not a quick process

Reply:
This is it https://github.com/eclipse/che/issues/22260
Then, I'm merging this PR and commenting in the issue, that we need to update the action when the issue done.